crm_multicompany_reporting_currency/models/crm_lead.py

```diff
@@ -81,3 +91,33 @@ def _compute_amount_multicompany_reporting_currency(self):
             else:
                 to_amount = record.expected_revenue * record.currency_rate
             record.amount_multicompany_reporting_currency = to_amount
+
+    @api.model
+    def read_group(self, domain, fields, groupby, **kwargs):
+        # Override read_group to calculate the sum of the
+        # expected revenue in current company currency
+        current_company = self.env.company
+        result = super().read_group(
```

Comment:
If we use `read_group` to calculate the sum, why do we use `search()` and `sum()` then? Can't we try with `read_group` aggregation?